enhancements/dns/plugins.md

```diff
@@ -186,11 +209,11 @@ apiVersion: v1
 data:
   Corefile: |
     foo.com:5353 {
-        forward . 1.2.3.4 5.6.7.8:5353
+        forward . 1.1.1.1 2.2.2.2:5353
         policy round_robin
     }
-    bar.com:5353 {
-        forward . 4.3.2.1 8.7.6.5:5353
+    bar.com:5353 example.com:5353 {
+        forward . 3.3.3.3 4.4.4.4:5454
         policy sequential
```

Comment:
Similar to the above, these two lines need to be changed to the following:
```
        forward . 3.3.3.3 4.4.4.4:5454 {
            policy sequential
        }
```

Reply:
Thanks for catching this. I'm going to drop `policy` per your feedback in https://github.com/openshift/enhancements/pull/68#discussion_r337136224